xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app-gpu-passthru">
+ <title>Configuring &gpuback; for &nvidia; cards</title>
+ <sect1 xml:id="gpu-passthru-intro">
+  <title>Introduction</title>
+
+  <para>
+   This article describes how to assign an &nvidia; GPU graphics card on the
+   host machine to a virtualized guest.
+  </para>
+ </sect1>
+ <sect1 xml:id="gpu-passthru-prerequisites">
+  <title>Prerequisites</title>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     GPU pass-through is supported on the &x86-64; architecture only.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     The host system needs to SLES 12 SP3 or newer.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     This article deals with a set of instructions based on V100/T1000 &nvidia;
+     cards, and is meant for GPU computation purposes only.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Verify that you are using an &nvidia; Tesla product&mdash;Maxwell, Pascal,
+     or Volta.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     To be able to manage the host system, you need an additional display card
+     on the host that you can use when configuring the &gpuback;, or a
+     functional SSH environment.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="gpu-passthru-host">
+  <title>Configuring the host</title>
+
+  <sect2 xml:id="gpu-passthru-verify-host">
+   <title>Verify the host environment</title>
+   <procedure>
+    <step>
+     <para>
+      Verify that the host system is SLES 12 SP3 or newer:
```

Comment:
```suggestion
      Verify that the host operating system is SLES 12 SP3 or newer:
```

Reply:
I am clarifying this because in context I would assume "host system" to mean the server hardware.